FreeRTOS-Plus/Demo/FreeRTOS_Plus_PKCS11_Windows_Simulator/examples/objects.c

```diff
@@ -0,0 +1,405 @@
+/*
+ * FreeRTOS PKCS #11 V1.0.3
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "task.h"
+
+/* Standard include. */
+#include "stdio.h"
+
+/* PKCS #11 includes. */
+#include "iot_pkcs11_config.h"
+#include "iot_pkcs11.h"
+#include "pkcs11.h"
+
+/* mbed TLS includes. */
+#include "mbedtls/pk.h"
+
+/* Demo includes. */
+#include "demo_helpers.h"
+#include "pkcs11_demos.h"
+
+/* RSA private key that has been generated off the device.
```

Comment:
I'd add some disclaimer like "users are not advised to hardcode keys, put keys in readable memory, ...".

You've made the point that "it's an example" and just for demonstration when "the device itself cannot create credentials". Though, in production, this approach is absolutely not OK in my opinion. And if anything, I'd like to make sure it doesn't come back to us.